src/resources/Pipelines/Statements/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './Statements';
```

Comment:
Don't forget to add `export * from ./Statements;` in https://github.com/coveo/platform-client/blob/5ffde1af5820099fe7eca2c32b3776587eb13a01/src/resources/Pipelines/index.ts too